assets/css/main.css

```diff
@@ -106,6 +106,7 @@ body {
 }
 .work {
   background: #000;
+  background-size: 100%;
```

Comment:
@jimijack I don't think we should use one huge background image for this. It should be broken into sections